library/src/main/java/me/toptas/fancyshowcase/FancyImageView.java

```diff
@@ -127,15 +151,17 @@ private void drawCircle(Canvas canvas) {
      * @param canvas canvas to draw
      */
     private void drawRoundedRectangle(Canvas canvas) {
-        float width = mCalculator.roundRectWidth();
         float left = mCalculator.roundRectLeft();
         float top = mCalculator.roundRectTop(mAnimCounter, mAnimMoveFactor);
         float right = mCalculator.roundRectRight();
         float bottom = mCalculator.roundRectBottom(mAnimCounter, mAnimMoveFactor);
-        canvas.drawRect(left, top, right, bottom, mErasePaint);
-        canvas.drawCircle(left, mCalculator.getCircleCenterY(),
-                mCalculator.roundRectLeftCircleRadius(mAnimCounter, mAnimMoveFactor), mErasePaint);
-        canvas.drawCircle(left + width, mCalculator.getCircleCenterY(),
-                mCalculator.roundRectLeftCircleRadius(mAnimCounter, mAnimMoveFactor), mErasePaint);
+
+        rectF.set(left, top, right, bottom);
+        canvas.drawRoundRect(rectF, mRoundRectRadius, mRoundRectRadius, mErasePaint);
+
+        mPath.reset();
+        mPath.moveTo(mCalculator.getCircleCenterX(), mCalculator.getCircleCenterY());
+        mPath.addRoundRect(left, top, right, bottom, mRoundRectRadius,mRoundRectRadius, Path.Direction.CW);
+        canvas.drawPath(mPath, mCircleBorderPaint);
```

Comment:
Bordering is not mandatory. So `mFocusBorderSize` default value should be 0. And border should be drawn if border size is bigger than 0. 
Please draw the border if border size is bigger than 0.